clowdapp.yaml

```diff
@@ -369,6 +369,66 @@ parameters:
   - name: IMAGE
     description: image name
     value: quay.io/cloudservices/historical-system-profiles-backend
+  - name: POPULATOR_CPU_LIMIT
+    description: cpu limit for service
+    value: 300m
+  - name: POPULATOR_MEMORY_LIMIT
+    description: memory limit for service
+    value: 1Gi
+  - name: POPULATOR_CPU_REQUESTS
+    description: cpu requests for service
+    value: 50m
+  - name: POPULATOR_MEMORY_REQUESTS
+    description: memory requests for service
+    value: 512mi
```

Comment:
I guess this should be `512Mi`. See `The ClowdApp "historical-system-profiles" is invalid: spec.jobs[0].podSpec.resources.requests.memory: Invalid value: "512mi": spec.jobs[0].podSpec.resources.requests.memory in body should match '^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$'`in https://ci.ext.devshift.net/job/RedHatInsights-historical-system-profiles-backend-pr-check/580/console